packages/ensnode-sdk/src/client.ts

```diff
@@ -0,0 +1,231 @@
+import type { Address } from "viem";
+import type { CoinType, Name } from "./utils/index.js";
+
+/**
+ * Default ENSNode API endpoint URL
+ */
+export const DEFAULT_ENSNODE_API_URL = "https://api.alpha.ensnode.io" as const;
+
+/**
+ * Configuration options for ENSNode API client
+ */
+export interface ClientOptions {
+  /** The ENSNode API endpoint URL */
+  endpointUrl: URL;
```

Comment:
i'd suggest renaming this to `url` for simplicity, as the user is already in the context of ENSNode (so `ensNodeUrl` is too verbose) and `endpointUrl` is redundant